src/utils/axes.ts

```diff
@@ -0,0 +1,26 @@
+import { axisLeft, axisBottom } from 'd3-axis';
+import { select } from 'd3-selection';
+import { scaleLinear } from 'd3-scale';
+
+export function drawYAxis(surface, size, border, dataMin, dataMax, ticks) {
+  let scale = scaleLinear().domain([dataMin, dataMax]).range([0, size.height]);
+  let translate = `translate(${border}, 0)`;
+  drawAxis(surface, size, axisLeft, scale, translate, 'y_axis', ticks);
+}
+export function drawXAxis(surface, size, border, dataMin, dataMax, ticks) {
+  let scale = scaleLinear().domain([dataMin, dataMax]).range([size.width, 0]);
+  let translate = `translate(${border}, ${size.height})`;
+  drawAxis(surface, size, axisBottom, scale, translate, 'x_axis', ticks);
+}
+
+export function drawAxis(surface, size, axis, scale, translate, id, ticks) {
+  let width = size.width,
+    height = size.height;
+
+  let svg = select(surface).attr('width', width).attr('height', height);
+
+  let a = axis().scale(scale);
+  ticks != null ? a.ticks(ticks, 'f') : null;
+
+  svg.select(`#${id}`).node() ? svg.select(`#${id}`).call(a) : svg.append('g').attr('transform', translate).attr('id', id).call(a);
```

Comment:
Turn this into an if statement for clarity. Ternary statements are mainly for returning values